wdl-lint/src/rules/input_not_sorted.rs

```diff
@@ -220,6 +220,10 @@ impl Rule for InputNotSortedRule {
             SyntaxKind::InputSectionNode,
         ])
     }
+
+    fn related_rules(&self) -> &[&'static str] {
+        &["ImportSort", "SectionOrdering"]
```

Comment:
I'm not certain these should be linked here. This seems redundant with the `sorting` tag. Also `ImportSort` is alphabetical, whereas `SectionOrdering` and `InputNotSorted` are custom sorting logic. So I wouldn't expect `ImportSort` to co-occur with the others.

Reply:
That seems fair to me. I'm fine removing these. But realizing that instead we should link `MatchingParameterMeta` here